spec/system/schools/advice_pages/meter_breakdown_spec.rb

```diff
@@ -23,6 +23,21 @@
   end
 end
 
+shared_examples 'the meter breakdown is not yet available' do
+  it 'shows not enough data message' do
+    visit path
+    expect(page).to have_content(I18n.t('advice_pages.not_enough_data.title'))
+  end
+end
+
+shared_examples 'the monthly chart is not available' do
+  it 'does not include the chart' do
+    visit path
```

Comment:
Very small thing but I'd put the `visit path` and the `click_on 'Analysis'` in a before block. Same with the one above with the `visit path`. Appreciate it's a one test shared example though so see why you might have done it.